app/views/api/v2/search/_base.json.jbuilder

```diff
@@ -1,9 +1,30 @@
-json.id result['timdex_record_id']
+json.timdex_record_id result['timdex_record_id']
```

Comment:
@hakbailey I seem to recall having a conversation around wanting to retain `id` as the user facing version of of `timdex_record_id`. Did we agree that `id` was fine for `timex-api` even if the data model wanted to use `timdex_record_id` or was there a strong desire to change what we refer to the id as outwardly?

Reply:
This is a case where I wish we could get some user feedback because I'm not sure what the best answer is. `timdex_record_id` is certainly more clear and accurate, especially given that most if not all records will have other IDs (DOIs, ISBNs, OCLCs, etc.) in the `identifiers` field, and those will have a `kind` value to indicate what kind of ID they are. Having an additional `id` field with no indication of what kind of ID it is seems potentially confusing, although we can and should document it in the spec. But I can also see an argument that `id` is a generally well-understood term when dealing with records of all sorts, and users might expect that. Do other folks have strong feelings one way or another?

Reply:
I think it's pretty common practice to have `id` signify the internal id for whatever you are working with. If we were going to allow multiple IDs in a single route, we'd want to clarify.

For instance `/api/v1/record/:id` is super clear and `/api/v1/record/:timdex_record_id` doesn't feel more clear as all it does is repeat information that is already implied (I'm using timdex and want a specific record by id).

Reply:
I suggest for now we move back to `id` for this and include evaluating this as part of our initial pathfinder work when we reach out to DLS/Ben. Until we publish v2 fully it will remain easy to swap this out if we find this is confusing.

Reply:
I suggest for now we move back to `id` for this and include evaluating this as part of our initial pathfinder work when we reach out to DLS/Ben. Until we publish v2 fully it will remain easy to swap this out if we find this is confusing.